src/Components/Record.php

```diff
@@ -22,6 +22,20 @@ public function __construct(array $dbInfo = null)
             $dbInfo = [];
         }
 
+        // Get database info from the configuration if available
```

Comment:
Thanks for the thorough description :)  In a perfect world I would implement option 3, however that will take more resources from us and the community devs then I see as worth it right now.  We've already released Blesta v5 and this update resolves some issues with the drawback being a small amount of bloat in the code. In a practical sense, those that don't use it will be unaffected.  I'll create an issue/task for these other possibilities as a future consideration.

Reply:
Yup, technical debt usually takes some time. You should still set `use Minphp\Configure\Configure;` at the top of the file though, since you're using `Configure::get`.

Reply:
It's using the Configure class from the bridge, not Minphp\Configure\Configure.